src/components/Image.tsx

```diff
@@ -0,0 +1,15 @@
+import NextImage from "next/image";
+
+// opt-out of image optimization, no-op
+const customLoader = ({ src }) => {
+  return src
+}
+
+export default function Image(props) {
```

Comment:
I think we could use https://github.com/vercel/next.js/blob/canary/packages/next/client/image.tsx#L90 for props type checking here